### ISSUE TITLE
No app notifications sent

### ISSUE THREAD

**Issue by ktpx** [hauckwill/linconnect-server#91]
Client never sends anything (android 7.1).  Test notifications work, but nothing is ever passed on from app notifications (app has permission enabled).

**Comment by freistil90** [comment]
Did you solve the issue? I have the same problem.

**Comment by ktpx** [comment]
No, I haven't bothered with it  as it seems this project is dead.
